distribution/solr-distro/pom.xml

```diff
@@ -83,9 +83,7 @@
                               ${solr.download.url}/lucene/solr/${solr.version}/solr-${solr.version}.zip
                             </url>
                             <unpack>true</unpack>
-                            <failOnError>true</failOnError>
```

Comment:
✏️ I don't think this is the correct approach, mostly because that version (9.0.0) is not currently showing up in the [Apache archives](https://archive.apache.org/dist/lucene/solr/).

Per Solr [documentation here](https://solr.apache.org/downloads.html#past-versions), I believe you need to change this URL to `https://archive.apache.org/dist/solr/solr/`.

Reply:
Oh. And it looks like they might just have gzipped tars there and not zips.

Reply:
DDF 2.26.x is still on Solr 8.6.1 and I don't think that URL has changed
https://archive.apache.org/dist/lucene/solr/8.6.1/solr-8.6.1.zip

I'm pretty sure the issue is that we have a bad artifact cached on Jenkins.  After this latest change the build failed trying to unzip the solr distro.

Reply:
1. Doh. Didn't look to see that this was going to the 2.26 branch. :)
2. Does sound like a bad artifact cached; I just downloaded the zip manually and checked its hash and it's correct.
3. I would still try to fix the underlying problem rather than removing the security check, especially as the security check is correctly telling you the file is wrong.